composer.json

```diff
@@ -35,20 +33,20 @@
         "fideloper/proxy": "^4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0",
+        "phpunit/phpunit": "^8.0|^9.0",
         "dms/phpunit-arraysubset-asserts": "^0.1.0",
-        "illuminate/database": "^6.0",
-        "illuminate/filesystem": "^6.0",
-        "illuminate/queue": "^6.0",
-        "illuminate/support": "^6.0",
-        "illuminate/log": "^6.0",
-        "illuminate/mail": "^6.0",
-        "illuminate/events": "^6.0",
-        "illuminate/pagination": "^6.0",
-        "illuminate/routing": "^6.0",
-        "illuminate/validation": "^6.0",
-        "illuminate/view": "^6.0",
-        "illuminate/cache": "^6.0",
+        "illuminate/database": "^6.11",
+        "illuminate/filesystem": "^6.11",
+        "illuminate/queue": "^6.11",
+        "illuminate/support": "^6.11",
+        "illuminate/log": "^6.11",
+        "illuminate/mail": "^6.11",
+        "illuminate/events": "^6.11",
+        "illuminate/pagination": "^6.11",
+        "illuminate/routing": "^6.11",
+        "illuminate/validation": "^6.11",
+        "illuminate/view": "^6.11",
+        "illuminate/cache": "^6.11",
```

Comment:
@bennothommo isn't it 6.0 that's the LTS?

Reply:
@LukeTowers the entire v6 line is LTS, because of their new semantic versioning policy. I'm  trying to ensure we're using the latest release and testing against that.